.github/ISSUE_TEMPLATE/bug_pythonmonkey.yaml

```
@@ -0,0 +1,80 @@
+name: File a Bug Report for PythonMonkey
+description: Use this template to report PythonMonkey-related issues. Thank you so much for making an issue!
+body:
+  - type: dropdown
+    id: issue-type
+    attributes:
+      label: Issue type
+      description: What type of issue would you like to report?
+      multiple: false
+      options:
+        - Bug
+        - Build/Install
+        - Performance
+        - Support
+        - Documentation Bug / Error
+        - Other
+    validations:
+      required: true
+
+  - type: dropdown
+    id: source
+    attributes:
+      label: How did you install PythonMonkey?
+      options:
+        - Source
+        - Installed from pip
+        - Other (Please specify in additional info)
+
+  - type: input
+    id: OS
+    attributes:
+      label: OS platform and distribution
+      placeholder: e.g., Linux Ubuntu 22.04
+
+  - type: input
+    id: Python
+    attributes:
+      label: Python version (`python --version`)
+      placeholder: e.g., 3.9
+
+  - type: input
+    id: PythonMonkey
+    attributes:
+      label: PythonMonkey version
+      placeholder: 0.0.1.dev997+1eb883
```

```suggestion
      placeholder: 0.2.0 or 0.0.1.dev997+1eb883
```

Show what a released version number would like

---

Good idea!